packages/db/src/models/execution.ts

```diff
@@ -34,6 +34,28 @@ class ExecutionPgModel extends BasePgModel<PostgresExecution, PostgresExecutionR
       .merge()
       .returning('cumulus_id');
   }
+
+  /**
+   * Get workflow_name column values from the execution cumulus_id
+   *
+   * @param {Knex | Knex.Transaction} knexOrTrx -
+   *  DB client or transaction
+   * @param {Function} executionCumulusIds -
+   * single execution cumulus_id or array of exeuction cumulus_ids
+   * @returns {Promise<Array<number>>} An array of workflow names
+   */
+  async getWorkflowNamesFromExecutionCumulusIds(
+    knexOrTrx: Knex | Knex.Transaction,
+    executionCumulusIds: Array<string> | string
+  ): Promise<Array<string>> {
+    const executionCumulusIdsArray
+      = (typeof executionCumulusIds === 'string') ? [executionCumulusIds] : executionCumulusIds;
+    const executionWorkflowNames = await knexOrTrx(this.tableName)
```

Comment:
```suggestion
    const executionResults = await knexOrTrx(this.tableName)
```

Kind of a nitpick. These values are results from the DB query at this point, which are objects containing the workflow name and not just the workflow name

Reply:
Yep, already changed this. Let me know if you think it should be 'executionResults' rather than just 'executions'. (Or if we should just return the value directly since we're no longer transforming it)